packages/bodiless-layouts/src/meta/index.tsx

```diff
@@ -104,16 +104,16 @@ const withTerm = (cat: string) => (term: string):HOC => Component => (
   withMeta({ categories: { [cat]: [term] } })(Component)
 );
 /**
- * preserveMeta returns takes an hoc and returns another one that will apply the hoc but preserve
+ * preserveMeta returns takes an HOC and returns another one that will apply the HOC but preserve
  * theMeta data from the component.
- * @param hoc the hoc to wrap.
+ * @param hoc the HOC to wrap.
  */
 const perserveMeta = (hoc: HOC): HOC => Component => (
   withMeta(Component)(hoc(Component) as ComponentType<any>)
 );
 
 /**
- * withFacet is expect to be passed to an on function and takes a term and and hoc (using curring)
+ * withFacet is expect to be passed to an on function and takes a term and and HOC (using curring)
```

Comment:
please fix the typo in the row 116
```suggestion
 * withFacet is expect to be passed to an on function and takes a term and a HOC (using curring)
```

Reply:
✅ Fixed.